### PR TITLE
Add Mullvad Leta Auto Login setting

### LINKED ISSUE #153: Feature request: Support automatic login to Leta search
When cookies are cleared in the browser, you get logged out of Mullvad Leta search. This can be quite a hassle, especially for a search engine. It would be nice if the browser extension could also help with automatically logging the user in for these situations. This is relevant for users who often use private browsing mode, or who use Mullvad Browser, or similar setups that clear site data frequently.

A nice example is the extension for Kagi search: https://github.com/kagisearch/browser_extensions When using it, you are always logged into Kagi and you are able to search.

### PR DESCRIPTION
**Pull request by ruihildt**
Closes #153 

**Limitations**
- Doesn't work with containers
- Doesn't work with in Private Browsing Mode in Firefox (as implementation is different from TB/MB)
- If you keep your browser open for more than 24 hours, you need to login/logout again

### CODE REVIEW

File: src/helpers/leta.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      method: 'POST',
+      body: JSON.stringify({
+        account_number: account,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    const response = await fetch('https://api.mullvad.net/auth/v1/webtoken', requestData);
```
Comment (hankolsen) [inline src/helpers/leta.ts:28]: Be aware that fetch does not throw errors (like axios). Fetch only throws error when a network error occurs.
Instead you should check the boolean `response.ok`

**Comment by hankolsen** [comment]
The account number is stored in clear text in the extension storage.
I'm not sure how big of a threat that is, but it feels a bit unsecure.

**Comment by ruihildt** [comment]
> Unexpected behaviour if you logout on the leta website:
> 
>     1. I add my account number to the plugin
> 
>     2. Go to leta and I'm automatically logged in. +1
> 
>     3. On leta, click Logout
> 
>     4. The plugin still has the account number and a Log out button

Changed the button label to "Save" and "Delete".

**Comment by ruihildt** [comment]
> The account number is stored in clear text in the extension storage.
> I'm not sure how big of a threat that is, but it feels a bit unsecure.

Nothing we can do, same as the Mullvad VPN app. The accoutn number is also visible in the extension interface.

**Review by hankolsen — APPROVED** [review]

**Pull request merged**